examples/wordcloud_cn.py

```diff
@@ -32,10 +33,11 @@
 imgname1 = d + '/wc_cn/LuXun.jpg'
 imgname2 = d + '/wc_cn/LuXun_colored.jpg'
 # read the mask / color image taken from
-back_coloring = imread(path.join(d, d + '/wc_cn/LuXun_color.jpg'))
+back_coloring = imread(d + '/wc_cn/LuXun_color.jpg')
 
 # Read the whole text.
-text = open(path.join(d, d + '/wc_cn/CalltoArms.txt')).read()
+with open(d + '/wc_cn/CalltoArms.txt', encoding='utf-8') as f:
```

Comment:
same here.

Reply:
'path.join' is proper, and 'encoding='utf-8' is necessary.
In python help file, the parameter is platform dependent. If I run the python file in Windows with out 'encoding='utf-8', return error
> encoding is the name of the encoding used to decode or encode the file. This should only be used in text mode. The default encoding is platform dependent (whatever locale.getpreferredencoding() returns)

Reply:
So do you want to change to join again?

Reply:
yes, 'path.join' is better